appstudio-controller/controllers/appstudio.redhat.com/applicationsnapshotenvironmentbinding_controller.go

```diff
@@ -272,6 +274,15 @@ func generateExpectedGitOpsDeployment(component appstudioshared.ComponentStatus,
 		}
 	}
 
+	res.ObjectMeta.Labels = make(map[string]string)
+
+	// Append ASEB labels with prefix "appstudio.openshift.io" to the gitopsDeployment labels
+	for bindingKey, bindingLabels := range binding.Labels {
```

Comment:
`bindingLabels` can be, `bindingValue` (or `bindingLabelValue` if you prefer to keep label in the name)